content/conditions/cold/manifest.json

```diff
@@ -7,14 +7,81 @@
   "content": {
     "header": [
       {
-        "type": "markdown",
-        "value": "You can often treat a cold without seeing your GP. You should begin to feel better in about a week or two."
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "You can often treat a cold without seeing your GP. You should begin to feel better in about a week or two."
+        }
       }
     ],
     "main": [
       {
-        "type": "markdown",
-        "value": "!file=main-content.md"
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "!file=main-content-1.md"
+        }
+      },
+      {
+        "type": "reveal",
+        "props": {
+          "summary": "Telling the difference between cold and flu",
+          "variant": "block",
+          "children": [
+            {
+              "type": "text",
+              "props": {
+                "variant": "markdown",
+                "value": "!file=reveal.md"
+              }
+            }
+          ]
+        }
+      },
+      {
+        "type": "text",
+        "props": {
+          "variant": "markdown",
+          "value": "!file=main-content-2.md"
+        }
+      },
+      {
+        "type": "callout",
+        "props": {
+          "variant": "info",
+          "compact": true,
+          "children": [
+            {
+              "type": "text",
+              "props": {
+                "variant": "markdown",
+                "value": "!file=callout-service-link.md"
```

Comment:
Yes, absolutely. I think once we understand a bit more about what services we provide the option to link to we can create a better structure for that component. 

There's some work being done about how that visually looks too but we may have variants that include linking to the new finders work from Leeds or linking to existing Choices services.

Think we should do that as a separate change though once we introduce that new component.